packages/react/src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React, { useRef, createRef } from 'react';
+import PropTypes from 'prop-types';
+import { ArrowRight20, ArrowDown20, Pdf20 } from '@carbon/icons-react';
```

Comment:
After removing the count does it make sense to also remove the restrictions on the specific icons that can be used within this component, and move that out into the leadspace pattern?

Reply:
I'm actually thinking of cleaning up the options here, because there is a `renderArrow` option here as well. 

I'm wondering if there is a restriction on icons for a reason, and perhaps we want to give them only a few options. Perhaps a question for @wonilsuhibm @oliviaflory @larahanlon2 .

cc: @annawen1

Reply:
After lengthy discussion, made this button group component more flexible, and allowing the parent pattern to do their own enforcement.